pydocstringformatter/formatting/formatter.py

```diff
@@ -88,7 +88,7 @@ def _treat_string(
                 return tokeninfo.string
             # If second line is empty we're dealing with a summary
             if lines[1] == "":
-                if lines[0][-1] != ".":
+                if lines[0][-1] not in {".", "?", "!", "‽"}:
```

Comment:
Should this be a constant ? ``END_OF_SENTENCE_PUNCTUATION`` ?

Reply:
Probably 😓  Could you make a quick PR?